src/libstd/collections/hash/map.rs

```diff
@@ -1282,7 +1291,7 @@ pub struct IterMut<'a, K: 'a, V: 'a> {
 /// HashMap move iterator.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct IntoIter<K, V> {
-    inner: table::IntoIter<K, V>,
+    pub(super) inner: table::IntoIter<K, V>,
```

Comment:
Hm for now instead of `pub(super)` could this just be a public function in this module to get access to the table? We could then  avoid exporting that as part of the `hash_map` module.